spec/models/mixins/supports_feature_mixin_spec.rb

```diff
@@ -267,7 +267,7 @@ def initialize(options = {})
     it "can be supported by the class" do
       stub_const("NukeablePost", Class.new(SpecialPost) do
         supports :nuke do
-          unsupported_reason_add(:nuke, "dont nuke the bribe") if bribe
+          unsupported_reason_add(:nuke, "do not nuke the bribe") if bribe
```

Comment:
WAT

Reply:
all kinds of interesting things in that file, yup